user/templates/base.html

```diff
@@ -6,19 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     
     <link rel="stylesheet" href="{% static 'styles.css' %}">
+    {% block stylesheets %}{% endblock stylesheets %}
     <title>Upcoming Shows</title>
 </head>
 <body>
-    <nav>
-        
-        <a class="nav-item nav-link" href="{% url 'user:shows' %}">Shows</a>
-        {% if request.user.is_staff %}
-        <a class="nav-item nav-link" href="{% url 'user:add_movie' %}">Add Movie</a>
-        {% endif %}
-        <a class="nav-item nav-link" href="{% url 'user:movie_list' %}">Movie List</a>
-    </nav>
+    <header>
+        <div class="logo">Movie Screen</div>
+        <nav>
+            {% if request.user.is_authenticated %}
+            {% if request.user.is_staff %}
+            <a class="nav-item nav-link " href="{% url 'user:add_movie' %}">Add Movie</a>
+            {% endif %}
```

Comment:
Can you add a nav-item for add-show as well ? I'll give you bonus

Reply:
okay let me see, but that will make the navbar look a bit cluttered